packet_unpacker.go

```diff
@@ -107,12 +107,8 @@ func (u *packetUnpacker) parseIETFFrame(r *bytes.Reader, typeByte byte, hdr *wir
 			err = qerr.Error(qerr.InvalidWindowUpdateData, err.Error())
 		}
 	case 0x6:
-		// TODO(#964): remove STOP_WAITING frames
 		// TODO(#878): implement the MAX_STREAM_ID frame
-		frame, err = wire.ParseStopWaitingFrame(r, hdr.PacketNumber, hdr.PacketNumberLen, u.version)
-		if err != nil {
-			err = qerr.Error(qerr.InvalidStopWaitingData, err.Error())
-		}
+		fallthrough
```

Comment:
How is falling through correct here? Should we error instead?

Reply:
I'll just remove this case (which is what we do for all other not yet implemented frame types).